src/index.html

```diff
@@ -294,26 +294,23 @@ <h2>week of ...</h2>
           </div>
         </section>
       </section>
-      <section class="single-recipe-view hidden">
-        <h1>Recipe Name</h1>
-        <div>
-          <button type="button" class="single-recipe-view__add-to-favorites-button">♡</button>
-          <p>ADD TO FAVORITES</p>
-        </div>
-        <div>
-          <button type="button" class="single-recipe-view__add-to-week-button"></button>
-          <p>ADD THIS WEEK'S RECIPES</p>
-        </div>
-        <p class="single-recipe-view__total-cost">💲COST: </p>
-        <img src= class="single-recipe-view__recipe-image" alt=>
-
-        <div class="single-recipe-view__ingredients">
-          <h2>INGREDIENTS</h2>
-          <div class="ingredients__ingredient-list"></div>
+      <section class="single-recipe-view hidden" id="singleRecipeView">
+        <div class="single-recipe-view__recipe-information">
+          <div class="recipe-information__container">
+            <h1 class="recipe-title" id="recipeName">Recipe Name</h1>
+            <button type="button" class="recipe-button add-to-favorites-button">♡ ADD TO FAVORITES</button>
+            <button type="button" class="recipe-button add-to-week-button">🗓 ADD TO THIS WEEK'S</button>
```

Comment:
Up to you, but I _think_ Juliana added the Google Font Icons library to your repo, which means you could use [this for the heart icon](https://fonts.google.com/icons?selected=Material%20Icons%20Outlined%3Afavorite_border%3A) and [this for the calendar icon](https://fonts.google.com/icons?selected=Material%20Icons%20Outlined%3Aevent%3A) if you wanted.

Reply:
Oh, I didn't know this existed, amazing!